src/dds/rs-dds-device-proxy.cpp

```diff
@@ -211,28 +206,43 @@ dds_device_proxy::dds_device_proxy( std::shared_ptr< context > ctx, std::shared_
 
     for( auto & sensor_info : sensor_name_to_info )
     {
+        LOG_DEBUG( sensor_info.first );
         sensor_info.second.proxy->initialization_done();
 
         // Set profile's ID based on the dds_stream's ID (index already set). Connect the profile to the extrinsics graph.
         for( auto & profile : sensor_info.second.proxy->get_stream_profiles() )
         {
-            sid_index type_and_index( profile->get_stream_type(), profile->get_stream_index());
+            if( auto p = std::dynamic_pointer_cast< librealsense::video_stream_profile_interface >( profile ) )
+            {
+                LOG_DEBUG( "    " << get_string( p->get_stream_type() ) << ' ' << p->get_stream_index() << ' '
```

Comment:
I think that framerate should follow the resolution and format after.
E.g. 1280x720@30 RGB8 (instead of 1280x720 RGB8 @ 30)

Reply:
OK :)